spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -446,6 +495,20 @@ public void setupMessageListener(Object messageListener) {
 	protected void validateConfiguration() {
 	}
 
+	protected void initializeProxy(Object delegate) {
+		if (this.getAdviceChain().length == 0) {
+			return;
+		}
+		ProxyFactory factory = new ProxyFactory();
+		for (Advice advice : getAdviceChain()) {
+			factory.addAdvisor(new DefaultPointcutAdvisor(Pointcut.TRUE, advice));
+		}
+		factory.setProxyTargetClass(false);
```

Comment:
It is `false` by default.

And I pointed previously that we don't need to specify `Pointcut.TRUE` explicitly. It is implied by the second `DefaultPointcutAdvisor` ctor.

Right, minor, but still hurts my eyes :smile: